src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +791,70 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    def display_benchmark_metrics(self):
+        """Display Benchmark Metrics for deployment configs."""
+        config_names = []
+        latencies = []
+        through_puts = []
+        instance_types = []
+        instance_rates = []
+        for deployment_config in self.list_deployment_configs():
+            config_names.append(deployment_config.get("ConfigName"))
+            instance_types.append(deployment_config.get("BenchmarkMetrics").key()[0])
```

Comment:
Is there a chance that a deployment config does not have a benchark metric? In which case do we need validation here?